public/resources/ts/calculate-player-stats.ts

```diff
@@ -127,6 +127,16 @@ export function getPlayerStats() {
     }
   }
 
+  {
```

Comment:
if everything is inside an `if` you dont need the outer `{}` scope, the `if` itself is the scope